media/css/debuggerPanel.css

```diff
@@ -45,3 +45,23 @@ body,
   margin-right: 16px;
   visibility: hidden;
 }
+
+.render-indicator {
+  display: none;
+  align-items: center;
+  justify-content: center;
```

Comment:
```suggestion
```
Those only have an effect if `display: flow;`.

Reply:
The style of the element will be changed to display flex during the execution of the Export.